src/doc/rustdoc/src/how-to-write-documentation.md

```diff
@@ -62,6 +133,21 @@ for argument in env::args() {
 [`args_os`]: ./fn.args_os.html
 ``````
 
+Everything before the first empty line will be reused to describe the component 
+in searches and module overviews.  For example, the function `std::env::args()`
+above will be shown on the [`std::env`] module documentation.  It is good
+practice to keep the summary to one line: concise writing is a goal of good
+documentation.
+
+Because the type system does a good job of defining what is passed to a function
+and what is returned from one, there is not a benefit of explicitly writing it
```

Comment:
```suggestion
Because the type system does a good job of defining what types a function
passes and returns, there is no benefit of explicitly writing it
```